fastlane/Fastfile

```diff
@@ -736,6 +738,7 @@ desc 'Build and upload DemoApp to Emerge'
 lane :emerge_upload do
   # Build and upload Release config
   build_ios_app(
+    export_method: 'ad-hoc',
```

Comment:
Can we remove this? I remember we tested this to fix Emerge, but this was not the issue, so better to revert this

Reply:
Actually, I think it's good to have it because the error message had much more sense when`export_method` was provided.